aws-android-sdk-mobile-client/src/androidTest/java/com/amazonaws/mobile/client/AWSMobileClientCustomAuthTest.java

```diff
@@ -216,8 +216,8 @@ public void onError(Exception e) {
     private void confirmSignIn() {
         final Map<String, String> res = new HashMap<String, String>();
         final Map<String, String> clientMetaData = new HashMap<String, String>();
-        clientMetaData.put("challengeResponseAnswer", "1134");
-        res.put(CognitoServiceConstants.CHLG_RESP_ANSWER, "1134");
+        clientMetaData.put("challengeResponseAnswer", "1133");
+        res.put(CognitoServiceConstants.CHLG_RESP_ANSWER, "1133");
```

Comment:
is this just some arbitrary value? Why not use a random number generator for this? 
Or use our `RandomString` utility from `testutils` package (if any string works)?

Reply:
This is the specific answer setup in the Lambda function.